### PR TITLE
[ README ] Update README file

### PR DESCRIPTION
**Pull request by jijoongmoon**
Update README file to give proper information about NNTrainer

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,39 +31,112 @@ Even though it trains part of the neural network models, NNtrainer requires quit
...
-- **_Model:_** Name of Model. Weight Data is saved in the name of this.
-- **_minibach:_** mini batch size
-- **_beta1,beta2,epsilon:_** hyper parameters for the adam optimizer
+ | Keyword | Layer Name | Description |
+ |:-------:|:---:|:---|
+ |  conv2d | Convolution 2D |Convolution 2-Dimentional Layer |
+ |  pooling2d | Pooling 2D |Pooling 2-Dimentional Layer. Support average / max / global average / global max pooing |
+ | flatten | Flatten | Flatten Layer |
+ | fully_connected | Fully Connected | Fully Connected Layer |
+ | input | Input | Input Layer.  This is not requies always. |
```
Comment (zhoonit) [inline README.md:44]:
```suggestion
 | input | Input | Input Layer.  This is not always required |
```

File: README.md [hunk 2]
```diff
@@ -31,39 +31,112 @@ Even though it trains part of the neural network models, NNtrainer requires quit
...
+ | batch_normalization | Batch Normalization Layer | Batch Normalization Layer. |
+ | loss layer | loss layer | hidden from users |
+ | activation | activaiton layer | hidden from users. but set by layer property |
```
Comment (zhoonit) [inline README.md:47]: Currently Activation layer is exposed to support activation after batch normalization

File: README.md [hunk 3]
```diff
@@ -31,39 +31,112 @@ Even though it trains part of the neural network models, NNtrainer requires quit
...
+### Examples for NNTrainer
+
+#### [Custom Shortcut Application](https://github.com/nnstreamer/nntrainer/tree/master/Applications/Tizen_native/CustomShortcut)
+
+
+This is demo application which enable user defined custom shortcut on galaxy watch.
+
+#### [MNIST Example](https://github.com/nnstreamer/nntrainer/tree/master/Applications/mnist)
+
+This is example to train mnist dataset. It consisits two convolution 2d layer, 2 pooling 2d layer, flatten layer and fully connected layer.
```
Comment (zhoonit) [inline README.md:119]:
```suggestion
This is example to train mnist dataset. It consists two convolution 2d layer, 2 pooling 2d layer, flatten layer and fully connected layer.
```

Comment (jijoongmoon) [reply README.md:119]: fixed.

File: README.md (hunk 2 above)
Comment (jijoongmoon) [reply README.md]: fixed.

File: README.md (hunk 1 above)
Comment (jijoongmoon) [reply README.md]: fixed.

**Pull request merged**